public/stylesheets/site/2.0/13-group-blurb.css

```diff
@@ -336,6 +336,11 @@ blurbs on the manage collection items pages, mostly reseting styles inherited fr
   clear: right;
 }
 
+.bookmark div.user {
+  box-sizing: border-box;
+  padding: 0 0.75em 0.5em;
```

Comment:
Could we use `0.429em 0.75em` (the value from `.blurb .blurb`, which gets applied on the tag bookmark pages) and move the padding rule to `.bookmark .user`? 

That way, if we ever want to tweak it, we'd only need to adjust it in one location to cover tag bookmark listings, user bookmark listings, bookmark listings for works (e.g., `/works/1073527/bookmarks`), and the list that appears when you press "Show Most Recent Bookmarks".

Reply:
Updated with your suggestion, thanks for taking the time to review!

In order to get that padding rule (`0.429em 0.75em`) to look right, I also needed to standardize the margin-top of the byline/heading on these bookmarks, since the tag bookmark listings had slightly less than the rest of the bookmark listings. I changed them all to have that smaller value from the tag bookmarks.